pkg/crc/machine/machine.go

```diff
@@ -157,13 +157,22 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		return *result, err
 	}
 
-	hostIP, err := network.DetermineHostIP(instanceIP)
+	var hostIP string
+	for i := 0; i < 30; i++ {
+		hostIP, err = network.DetermineHostIP(instanceIP)
+		if err == nil {
+			logging.DebugF("Bridge IP on the host: %s", hostIP)
+			break
+		} else {
+			logging.DebugF("Error finding host IP (%v) - retrying", err)
+		}
+		time.Sleep(2 * time.Second)
+	}
```

Comment:
@cfergeau `host.Driver.GetIP()` suppose to wait till the VM is up and then only provide the IP since the mac address which it looks suppose to be different for each run until we are passing a specific UUID => https://github.com/code-ready/machine-driver-hyperkit/blob/master/pkg/hyperkit/driver.go#L220

Reply:
See below for the explanation why we do not generate a new UUID each run (if the VM already exists, we do not recreate it with a different UUID)